charts/sequencer/values.yaml

```diff
@@ -26,6 +26,7 @@ genesis:
   genesisTime: ""  # '2023-09-22T17:22:35.092832Z'
   addressPrefixes:
     base: "astria"
+    ibcCompat: "astriacompat"
```

Comment:
do these necessarily have to be different, or is it just to easily differentiate between the two address formats?

Reply:
I think it's useful for easy identification of the two formats.

Reply:
Yes indeed. This is to make identification easier. We could always first try one and then the other format, but I think this is much cleaner.

Reply:
do we need to specify `ibcCompat` in the genesis or would it be sufficient to just append `compat` onto the base prefix?

Reply:
I am actually ambivalent here. I like to explicitly spell it out, but do you think there might be a use case where we send funds back to, e.g., an `"astriabridge"` prefixed address?